### PR TITLE
fix: [BB-334] crops info styling fixes

### PR DESCRIPTION
**Pull request by kcaparas1630**
# Description

Fixed crops info module styling to be almost exactly the same as the mockups.

## Type of changes

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have already been accepted and merged

## Further comments

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-sustainment-capstone-2024-338-frontend.apps.silver.devops.gov.bc.ca)
- [Backend](https://nr-sustainment-capstone-2024-338-backend.apps.silver.devops.gov.bc.ca/api/health)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-sustainment-capstone-2024/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-sustainment-capstone-2024/actions/workflows/merge.yml)

### CODE REVIEW

File: frontend/src/Commons/Forms/FormModule/FormModule.style.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ const StyledFormContent = styled.div`
   border-radius: 2px;
   border-top-left-radius: 1px;
   border-top-right-radius: 1px;
-  padding: 0 10px 15px 10px;
+  padding: 11px 10px 15px 10px;
```
Comment (kcaparas1630) [inline frontend/src/Commons/Forms/FormModule/FormModule.style.ts:29]: gives a bit of spacing at the top of form content

File: frontend/src/Commons/Forms/InputModules/Crops/CropsInfo.styles.ts [hunk 2]
```diff
@@ -4,9 +4,9 @@ import * as tokens from '@bcgov/design-tokens/js';
 
 const StyledCropsSmallGroup = styled.div`
   display: flex;
-  gap: 20px;
+  gap: 24px;
```
Comment (kcaparas1630) [inline frontend/src/Commons/Forms/InputModules/Crops/CropsInfo.styles.ts:7]: This is the spacing for vertical, horizontal might change. But vertical alignment is important to be 24px at all times.

File: frontend/src/Commons/Forms/InputModules/Crops/CropsInfo.tsx [hunk 3]
```diff
@@ -223,19 +221,21 @@ const CropsInfoComponent: FC<InputModuleProps> = ({
...
-                          desktopWidth="50%"
-                        />
-                        <p>tons/ac</p>
-                      </StyledAreaContainer>
+                      <CustomField
+                        label="Yield"
+                        id="yield"
+                        name="yield"
+                        type="number"
+                        isYield
```
Comment (kcaparas1630) [inline frontend/src/Commons/Forms/InputModules/Crops/CropsInfo.tsx:234]: quick response to editing a field and not affecting other fields using ternary operator

**Review by GDamaso — APPROVED** [review]
LGTM :rocket:

**Pull request merged**